src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/validation/AbstractConfigurationValidator.java

```diff
@@ -188,6 +189,24 @@ public boolean validate() {
             return false;
         }
 
+        //null element in the values of all the possible keys with DataType as ARRAY
+        JsonNode contentAsNode = getContentAsNode();
```

Comment:
It is not necessary to introduce local variable, use member variable directly.

Reply:
fixed